lib/doctor_teeth.rb

```diff
@@ -6,12 +6,15 @@ module DoctorTeeth
   require 'doctor_teeth/json_parser'
 
   # method stub for parsing
+  # @param [String] path of junit xml file(s) of test_run data
```

Comment:
Seems like this should take the xml as a string, not a path to the xml, but doesn't have to be fixed in this PR.

Reply:
at the time of this PR this actually was the junit xml file path.  in PR 7 it's actually the xml data.  i've got a ton of docs fixups coming because i added yardstick to check docs per-pr